### PR TITLE
headings and table sort

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Updating headings per sentence style and table sorting
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-155482](https://oktainc.atlassian.net/browse/OKTA-155482)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 1]
```diff
@@ -7,45 +7,44 @@ category: management
...
 
-All responses return the created [Application](#application-model).
 
-#### Add Bookmark Application
+All responses return the created [application](#application-model).
 
+#### Add Bookmark application
 
-Adds an new bookmark application to your organization
+Adds an new Bookmark application to your organization
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: Adds **a** new Bookmark... (not "an")

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 2]
```diff
@@ -129,7 +128,7 @@ curl -v -X POST \
 }
 ```
 
-#### Add Basic Authentication Application
+#### Add Basic Authentication application
 
 Adds an new application that uses HTTP Basic Authentication Scheme and requires a browser plugin
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: Adds **a** new applicaiton....

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 3]
```diff
@@ -1126,30 +1121,30 @@ Adds an OAuth 2.0 client application. This application is only available to the
...
-| policy_uri <ApiLifecycle access="ea" />     | URL string of a web page providing the client's policy document                                                                                                                                                            | URL                                                                                            | TRUE       | FALSE    | FALSE      |
-| consent_method <ApiLifecycle access="ea" /> | Indicates whether user consent is required or implicit. Valid values: `REQUIRED`, `TRUSTED`. Default value is `TRUSTED`                                                                                                    | String                                                                                         | TRUE       | FALSE    | TRUE       |
 
-###### Credentials Settings Details
+###### Credentials settings details
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: Could we just say "Credentials settings" - having three plurals in a row seems wrong somehow.

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 4]
```diff
@@ -2286,30 +2281,30 @@ curl -v -X GET \
...
+> **Note:** All properties must be specified when updating an app. **Delta updates are not supported.**
 
-##### Response Parameters
+##### Response parameters
 
-Updated [Application](#application-model)
+Updated [application](#application-model)
 
-#### Set SWA User-Editable UserName & Password
+#### Set SWA user-editable username & password
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: Replace & with "and".

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 5]
```diff
@@ -3410,13 +3405,13 @@ Assigns a user without a [profile](#application-user-profile-object) to an appli
 
 > **Note:** Only the user's ID is required for the request body of applications with [SignOn Modes](#signon-modes) or [Authentication Schemes](#authentication-schemes) that don't require or support credentials.
 
-> **Note:** If your SSO application requires a profile but doesn't have provisioning enabled, you should add a profile to the request and use the [Assign User to Application for SSO & Provisioning](#assign-user-to-application-for-sso--provisioning) operation.
+> **Note:** If your SSO application requires a profile but doesn't have provisioning enabled, you should add a profile to the request and use the [Assign user to application for SSO & provisioning](#assign-user-to-application-for-sso--provisioning) operation.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: Remove the "&" here and in the referenced heading

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 6]
```diff
@@ -3464,13 +3459,13 @@ curl -v -X POST \
 }
 ```
 
-### Assign User to Application for SSO & Provisioning
+### Assign user to application for SSO & provisioning
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: Replace "&" with "and"

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 7]
```diff
@@ -5159,52 +5155,52 @@ HTTP/1.1 204 No Content
...
+| visibility         | visibility settings for app                    | [Visibility Object](#visibility-object)                              | TRUE         | FALSE      | FALSE        |               |             |
 
 Property details
 
  * `id`, `created`, `lastUpdated`, `status`, `_links`, and `_embedded` are only available after an app is created.
  * `profile` is only available for OAuth 2.0 client apps. See [Profile Object](#profile-object).
  * When you specify a value for the `request_object_signing_alg` property, all request objects from the client are rejected if not signed with the specified algorithm. The algorithm must be used when the request object is passed by value (using the request parameter). If a value for `request_object_signing_alg` isn't specified, the default is any algorithm that is supported by both the client and the server.
 
-##### App Names & Settings
+##### App names & settings
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: Replace "&" with "and"

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 8]
```diff
@@ -5862,17 +5858,17 @@ The metadata for a CSR
 | Property               | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly | MinLength | MaxLength | Validation |
 | ----------------       | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- | ---------- |
 | commonName             | Common name of the subject                                   | String                                                                      | TRUE     | FALSE  | FALSE    |           |           |            |
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: Why is this one capitalized and the other descriptions are not?

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 1 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: GAH! I didn't even SEE that. LOL Thanks.

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 3 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: > Could we just say "Credentials settings" - having three plurals in a row seems wrong somehow.

I made it Details. There was a Credentials section and a Settings section. So, I made it the Details section.

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 8 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: missed one?

**Pull request merged**